conda/recipes/cuxfilter/recipe.yaml

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
+schema_version: 1
+
+context:
+  version: ${{ env.get("RAPIDS_PACKAGE_VERSION") }}
+  minor_version: ${{ (version | split("."))[:2] | join(".") }}
+  cuda_version: ${{ (env.get("RAPIDS_CUDA_VERSION") | split("."))[:2] | join(".") }}
+  cuda_major: '${{ (env.get("RAPIDS_CUDA_VERSION") | split("."))[0] }}'
+  date_string: '${{ env.get("RAPIDS_DATE_STRING") }}'
+  py_version: ${{ env.get("RAPIDS_PY_VERSION") }}
+  py_buildstring: ${{ py_version | version_to_buildstring }}
+  head_rev: ${{ git.head_rev(".")[:8] }}
+
+package:
+  name: cuxfilter
+  version: ${{ version }}
+
+source:
+  path: ../../..
+
+build:
+  string: cuda${{ cuda_major }}_py${{ py_buildstring }}_${{ date_string }}_${{ head_rev }}
+  script:
+    content: |
+      ./build.sh cuxfilter
+
+requirements:
+  host:
+    - cuda-version =${{ cuda_version }}
+    - python =${{ py_version }}
+    - rapids-build-backend >=0.3.0,<0.4.0.dev0
+    - setuptools
+  run:
+    - ${{ pin_compatible("cuda-version", upper_bound="x", lower_bound="x") }}
+    - cudf =${{ minor_version }}
+    - cupy >=12.0.0
+    - cuspatial =${{ minor_version }}
+    - dask-cudf =${{ minor_version }}
+    - datashader >=0.15
+    - geopandas >=0.11.0
+    # TODO: shapely 2.1.0 transitive dependency breaks geopandas
```

Comment:
Is this still true, or has it been solved? I think someone said a new geopandas version fixed this.

Reply:
heh, i think that was me.  I'll try removing it and see what happens